daemon/algod/api/algod.oas3.yml

```diff
@@ -3659,6 +3678,11 @@
                     "result": {
                       "description": "base64 encoded program bytes",
                       "type": "string"
+                    },
+                    "sourcemap": {
+                      "description": "JSON of the source map",
+                      "properties": {},
```

Comment:
can this be omitted if we dont specify any properties? or should we specify them?